docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,27 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | True if female                              | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`pflichtbeitragszeit`  | Months of mandtory contributions            | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`freiw_beitragszeit`   | Months of voluntary contributions           | float        |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit`     || Months of sickness, pregnancy/Mutterschutz,|| float       |
+||                        || unemployment, school/university after age  ||             |
+||                        || 17 (max 8 years), insurance retirement     ||             |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit_45`  || Months of sickness, unemployment (only time|| float       |
+||                        || of Entgeltersatzleistungen, not ALGII),    ||             |
+||                        || interim payments (Übergangsgeld), marginal ||             |
+||                        || employment                                 ||             |
++-------------------------+---------------------------------------------+--------------+
+|| _`ersatzzeit`          || Months during military, persecution/escape,|| float       |
+||                        || internment and consecutive sickness        ||             |
++-------------------------+---------------------------------------------+--------------+
+| _`kinder_berückz`       | Months of childcare till age 10             | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`pflege9295_berückz`   | Months of home care (01.01.1992-31.03.1995) | float        |
```

Comment:
```suggestion
| _`pflege9295_berücks_zeit`   | Months of home care (01.01.1992-31.03.1995) | float        |
```

Might be too long, but consistency across variables trumps anything here. Might need to shorten others.

Reply:
changed